src/audiomixerboard.cpp

```diff
@@ -309,6 +338,20 @@ void CChannelFader::SendFaderLevelToServer ( const int iLevel )
     }
 }
 
+void CChannelFader::SendPanValueToServer ( const int iPan )
+{
+
+    // if mute flag is set or other channel is on solo, do not apply the pan
```

Comment:
Why is this condition? If you do not send the pan to the server, the following could happen: The Mute is enabled. Then the user changes the pan which is not sent to the server. Then he unmutes. I assume the pan is not correct at the server then.

Reply:
Do you agree with my concern?